lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -1881,9 +1876,58 @@ GLStatus TFGraphLowering::visitGraphOperationInst(GraphOperationInst *inst) {
         break;
       }
       case SymbolicValue::Array: {
-        // FIXME: Handle array attributes.
+        CanType elementType;
+        auto rawElements = attrValue.getArrayValue(elementType);
+        SmallVector<SymbolicValue, 4> elements;
+        elements.reserve(rawElements.size());
+        for (auto elt : rawElements)
+          elements.push_back(elt.lookThroughSingleElementAggregates());
+
+        auto elementTypeString = elementType->getString();
+
+        // TODO: TF_SetAttrTypeList
+
+        if (elementTypeString == "String") {
+          SmallVector<const void*, 4> pointers;
+          SmallVector<size_t, 4> sizes;
```

Comment:
do we want to call reserve() for `pointers` and `sizes` as well? same for `values` below.

Reply:
Sure, done.